test/create-open.test.js

```diff
@@ -165,22 +165,21 @@ Object.keys(testAPIs).forEach(API => {
 
           it('creates an access controller and adds ourselves as writer by default', async () => {
             db = await orbitdb.create('fourth', 'feed')
-            assert.deepEqual(db.access.write, [orbitdb.key.getPublic('hex')])
+            assert.deepEqual(db.access.write, [orbitdb.identity.publicKey])
           })
 
           it('creates an access controller and adds writers', async () => {
-            db = await orbitdb.create('fourth', 'feed', { write: ['another-key', 'yet-another-key', orbitdb.key.getPublic('hex')] })
-            assert.deepEqual(db.access.write, ['another-key', 'yet-another-key', orbitdb.key.getPublic('hex')])
-          })
-
-          it('creates an access controller and doesn\'t add an admin', async () => {
-            db = await orbitdb.create('sixth', 'feed')
-            assert.deepEqual(db.access.admin, [])
+            db = await orbitdb.create('fourth', 'feed', { 
+              accessController: {
```

Comment:
Highlighting this to make sure we update the docs to reflect this change. Previously the users gave ac keys with `{ write: [...] }` and now they're under `accessController` field, so this is something the users will prolly have troubles with.